support-frontend/assets/helpers/productPrice/productOptions.ts

```diff
@@ -35,7 +35,7 @@ export type PaperProductOptions =
 	| typeof Sixday
 	| typeof Everyday;
 
-const ActivePaperProductTypes = [Everyday, Weekend, Saturday] as const;
+const ActivePaperProductTypes = [Everyday, Weekend, Saturday, Sunday] as const;
```

Comment:
Is it okay to add `Sunday` here? Any other effects I should be aware of?

Reply:
I think adding that will put Sunday back on sale as soon as it's merged, so make sure that's what you want!

Reply:
Ah okay thanks. I'll dig into exactly what that means in terms of how this is used.

Reply:
This was a good spot - I don't think it's what I want right now. `Sunday` shouldn't go back on sale when this PR is merged. I wonder if I can differentiate by stage or something, I'd like to be testing this in CODE at least and if `Sunday` isn't active I can't complete the checkout flow.

Reply:
I've made a change - we'll now skip over Sunday when rendering the cards on the paper landing page, but other than that Sunday is an active product type. I've added a story for `PaperProductPrices` which should catch any change which accidentally makes the Sunday card visible.